src/PP.Domain/Attractions/Attraction.cs

```diff
@@ -12,5 +12,11 @@ public class Attraction : AuditedAggregateRoot<Guid>
 
         public ICollection<RideRestriction> RideRestrictions { get; set; }
 
+        public Attraction()
```

Comment:
I added a constructor here to initialise the list to an empty List. This seems a bit fishy to me. There must be a better way, but we'll take this solution for now.